### ISSUE TITLE
Menu not showing on UBUNTU 14.04

### ISSUE THREAD

**Issue by odaibanet** [OpenShot/openshot-qt#161]
No menu shown 
![osve](https://cloud.githubusercontent.com/assets/18536331/14613041/0cbf98f8-0560-11e6-8219-57d14586b35d.png)

**Comment by jonoomph** [comment]
Does the menu show up at the top of the screen (when you move your mouse to the very top left of the screen)? See screenshot->
![openshot2xmenu](https://cloud.githubusercontent.com/assets/5551883/14656937/2315b0c4-0650-11e6-9bf5-82b2635ed5b5.png)

There are a few known bugs around the way Ubuntu extracts the main menu from Qt applications, and moves it to the top of the screen. Especially if you have disabled that functionality. Most likely this is not a bug I can fix in OpenShot, and it will require changing some settings on Ubuntu, or updating some package in Ubuntu to fix this issue.

**Comment by odaibanet** [comment]
Oh, I am not using the UNITY gui, I changed to XFCE instead so, is this possible to see the menu like this? Best Regards!

**Comment by tipichris** [comment]
try running openshot using a script like this
```
#!/bin/bash
export QT_QPA_PLATFORMTHEME=
/usr/bin/openshot-qt
```

**Comment by drexlar1** [comment]
tipichris, thank you for your wonderful suggestion to use your script. It solved the problem I had with OpenShot 2.0.7 in Ubuntu 14.04, that refused to show the menu.
